internal/versions/versions.go

```diff
@@ -266,6 +269,19 @@ func (h *handler) validateVersions() error {
 		if osImage.Version == nil {
 			return errors.Errorf(fmt.Sprintf(missingValueTemplate, "Version"))
 		}
+
+		missingValueTemplate = "Missing value in ReleaseImage for '%s' field"
+		for _, release := range h.releaseImages {
```

Comment:
I think this for loop can be outside of the current for loop scope

Reply:
right, done.